mass_mailing_custom_unsubscribe/README.rst

```diff
@@ -8,10 +8,16 @@ Customizable unsubscription process on mass mailing emails
 With this module you can set a custom unsubscribe link appended at the bottom
 of mass mailing emails.
 
+It also displays a beautiful and simple unsubscription form when somebody
+unsubscribes, to let you know why he left; and then displays to him a beautiful
```

Comment:
Don't use he/him. Better to use an inpersonal style. Remove that parts from the sentence. The meaning is the same.